### PR TITLE
Chapter 5

### PR DESCRIPTION
**Pull request by bast**
Work in progress ...

### CODE REVIEW

**Comment by bast** [comment]
Oops :-) Will fix in a moment.

**Comment by bast** [comment]
Hmm... why is Appveyor not running these tests?

**Comment by robertodr** [comment]
What?

**Comment by bast** [comment]
I am also pushing this as a branch to the central repo so we get testing but I keep this PR open and will not open another one.

File: chapter-05/recipe-08/c-example-3.5/CMakeLists.txt [hunk 1]
```diff
@@ -25,14 +26,19 @@ int main(int argc, char * argv[]) {
...
 set(CMAKE_REQUIRED_LIBRARIES ${UUID_LIBRARIES})
 check_c_source_runs("${_test_uuid}" _runs)
+unset(CMAKE_REQUIRED_INCLUDES)
+unset(CMAKE_REQUIRED_LIBRARIES)
+
 if(NOT _runs)
   message(FATAL_ERROR "Cannot run a simple C executable using libuuid!")
 endif()
 
 add_executable(use-uuid use-uuid.cpp)
```
Comment (robertodr) [inline chapter-05/recipe-08/c-example-3.5/CMakeLists.txt]: The source code is C++ but the example folder says C. Is this intended?

Comment (robertodr) [reply chapter-05/recipe-08/c-example-3.5/CMakeLists.txt]: Yes, it was much clearer after reading the text. I am merging this one in the hope that we get appveyor to run on `master` Can you look into Appveyor settings to see if anything changed?

Comment (robertodr) [reply chapter-05/recipe-08/c-example-3.5/CMakeLists.txt]: Ah, and can you also delete the branch?

**Pull request merged**